cdci_data_analysis/analysis/parameters.py

```diff
@@ -164,13 +164,29 @@ def __init__(self,
                  value=None,
                  units=None,
                  name: Union[str, None] = None,
+                 # TODO should we remove units/type/format knowledge from the Parameter class?
+"""
+format:
+   every parameter has a format. Format defines representation of the parameter value. 
+   Default format of  defines unique representation 
+  
+TODO
```

Comment:
```suggestion
format:
    * Every `Parameter` has a *format*. *Format* defines representation of the parameter value in parameter URL, analysis_parameters dictionary, oda_api, and embedded in fits files.       
    * Parameter dictionary may include format specifiers for each parameter.  If non-default format is used the parameter, parameter format specifier is required. If not specified, the default is used.
    * *default format* of defines **unique** representation of the parameter. Any parameter value can be converted to default *format*. By using default parameter representations, it is possible to construct unique and deterministic request parameter dictionaries, URLs, oda_api codes.

unit:
   * physical quantities are represented as floats (and should inherit from `Float`) and are scaled with *units*. The same value may be represented as different floats if units are correspondingly different.
   * units are treated similarly to formats: there are unit specifiers and default units. 
   
type:
   * each parameter is constructed with a value of some types. Types is verified at construction, and converted to default type. Further representations are defined by units and formats

  
TODO see through that this is implemented https://github.com/oda-hub/dispatcher-app/issues/339
```

Reply:
Does it make sense to add the following point to the unit part:

```
* for some parameters units might not be specified (eg Time) since not necessary, the same applies the other way around
```

Also, regarding the type: at the moment this might not be specified for certain parameters (eg Time, Name), shall we mention that?

Reply:
as we use astropy Time, it would not inherit from `Float`. And only `Float` can have units. 
In principle, for Time, this is not the only option. We could have used time since some epoch. This would be a float (measured in days, or seconds etc since epoch).
But, if it's astropy Time - it does not have inherit  from Float, and has no type. In principle time since epoch is more like TimeDelta than Time - timestamp.
I add this small example.

Time and Name would have type,  why not? astropy.time.Time and string correspondingly.

Reply:
> 
> Time and Name would have type, why not? astropy.time.Time and string correspondingly.

Ok then this will have to be considered when dealing with #339